README.md

```diff
@@ -24,10 +23,19 @@ A simple example extracted from the unit tests:
     jb.destroy();
     assertEquals(ibb, abb);
 ```
-Blosc shared library should be in PATH on Windows or in LD_LIBRARY_PATH on Linux/Unix.
+## Installation
+First of all, you need to install the Blosc library (visit https://github.com/Blosc/c-blosc for more details), in short, if you already have CMake, executing the following commands should do the work:
+```bash
+git clone https://github.com/Blosc/c-blosc.git
+cd c-blosc
+mkdir build
+cd build
+cmake -DDEACTIVATE_ZLIB=ON -DCMAKE_GENERATOR_PLATFORM=x64 ..
+cmake --build . --target install
+```
+Make sure Blosc shared library is in PATH on Windows (```copy "c:\Program Files (x86)\blosc\lib\blosc.dll" c:\Windows\System32```) or in LD_LIBRARY_PATH on Linux/Unix (```export LD_LIBRARY_PATH=/usr/local/lib```).
```

Comment:
`LD_LIBRARY_PATH` is normally used by developers only.  On Unix the user will just have to put the library in the system search path (typically '/usr/lib' or '/usr/lib64'). Please update that accordingly.